src/onboarding/steps.ts

```diff
@@ -0,0 +1,208 @@
+import { BIOMETRY_TYPE } from 'react-native-keychain'
+import { initializeAccount } from 'src/account/actions'
+import {
+  choseToRestoreAccountSelector,
+  recoveringFromStoreWipeSelector,
+} from 'src/account/selectors'
+import {
+  numberVerifiedCentrallySelector,
+  skipVerificationSelector,
+  supportedBiometryTypeSelector,
+} from 'src/app/selectors'
+import { setHasSeenVerificationNux } from 'src/identity/actions'
+import * as NavigationService from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+import { store } from 'src/redux/store'
+
+export const FIRST_ONBOARDING_SCREEN = Screens.NameAndPicture
+export const END_OF_ONBOARDING_SCREEN = Screens.WalletHome
+
+interface NavigatorFunctions {
+  navigate: typeof NavigationService.navigate | ((screen: Screens) => void)
```

Comment:
i don't really get the OR types here. why is that needed?

Reply:
No, the first type is needed for actually executing the navigation. The second type is needed for calculating the steps